src/dataviews/histogram-dataview-model.js

```diff
@@ -129,6 +129,10 @@ module.exports = DataviewModelBase.extend({
     };
   },
 
+  getGlobalDistributionType: function () {
+    return this.getDistributionType.call(this.getUnfilteredDataModel());
```

Comment:
shouldn't that method be in histogram model (that call changing the this sounds like a hack?)